packages/react/src/components/Modal/index.mdx

```diff
@@ -0,0 +1,256 @@
+import { Canvas, Meta, Story, Source, ArgTypes } from '@storybook/blocks';
+import * as Stories from './index.stories';
+import * as PreBuiltStories from './PreBuilt.stories';
+import { Modal } from './index';
+
+<Meta of={Stories}/>
+
+# Modal
+
+Cunningham provides a versatile Modal component for displaying any kind of information.
+
+<Canvas>
+  <Story of={Stories.ThreeButtons} inline={false}/>
+</Canvas>
+
+<Source
+  language='ts'
+  dark
+  format={false}
+  code={`import { Modal } from "@openfun/cunningham-react";`}
+/>
+
+> ⚠️ If you want to try dark theme on the modal, you need to go on individual stories. It will not work on this page due to iframe wrapping.
+
+## Usage
+
+The component is easy to use. You need to use the `useModal()` hook to get all the needed props and helper functions.
```

Comment:
IMO, we should add a paragrah to explain in detail the hook `useModal` and detail all stuff returned by it.